packages/ui-core/shared/src/lib/selectors/team/team/team.component.ts

```
@@ -181,153 +226,232 @@
+
+	/**
+	 * Handles changes in the selected organization.
+	 *
+	 * Updates the local organization property and triggers a team fetch.
+	 */
+	private initializeOrganizationSelection(): void {
+		this._store.selectedOrganization$
+			.pipe(
+				distinctUntilChange(),
+				filter((organization: IOrganization) => !!organization),
+				tap((organization: IOrganization) => (this.organization = organization)),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)